system/modules/core/classes/Theme.php

```diff
@@ -548,8 +546,8 @@ protected function extractThemeFiles($arrFiles, $arrDbFields)
 							}
 						}
 
-						// Adjust the file paths in style sheets
-						elseif (\Config::get('uploadPath') != 'files' && ($table == 'tl_style_sheet' || $table == 'tl_style') && strpos($value, 'files') !== false)
+						// Adjust the file paths in style sheets and tl_files
+						elseif (\Config::get('uploadPath') != 'files' && ($table == 'tl_style_sheet' || $table == 'tl_style' || ($table == 'tl_files' && $name == 'path')) && strpos($value, 'files') !== false)
```

Comment:
Why did you add an additional check for the field name here?

```
($table == 'tl_files' && $name == 'path'))
```

Are there other fields which potentially contain `files` and shall not be changed?